Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM golang:1.13-buster AS builder
 # Download tools
 RUN curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.17.1
 
+RUN wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.23.8
```

Comment:
Why is this adding a second copy of `golangci-lint`?

Reply:
oh woops I'll rip that out. I had some initial issues with the image and the included golangci-lint.